quarto/src/Controller/MainController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace App\Controller;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
```

Comment:
Did you install a Php linter? for example to follow the [PSR-2](https://www.php-fig.org/psr/psr-2/) standard. Someting like [PHP Coding Standards Fixer](http://cs.sensiolabs.org/) or [PHP Formatter](https://marketplace.visualstudio.com/items?itemName=Sophisticode.php-formatter).

Here for example it seems to me that an empty line is missing.